task/scripts/build.sh

```diff
@@ -120,3 +120,8 @@ providers {
 }
 
 EOF
+
+echo "Installing Bats BASH testing framework"
+git clone https://github.com/sstephenson/bats.git
+cd bats
```

Comment:
Can we use `pushd` and `popd` in case we install stuff after this later on?

Reply:
Updated